src/Microsoft.ML.SearchSpace/Schema/SchemaBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.ML.SearchSpace.Schema
+{
+    public abstract class SchemaBase
+    {
+        [JsonProperty(PropertyName = "schema_type", Required = Required.Always)]
+        public abstract SchemaType SchemaType { get; }
+
+        [JsonProperty(PropertyName = "version", Required = Required.Always)]
+        public abstract int Version { get; }
```

Comment:
What is the point of this `Version` property?

Reply:
It's saved for the possible schema change in the future, provides us with the chance to upgrade from old schema to newer one seamlessly.